src/Features/VisualBasic/Portable/CodeFixes/QualifyMemberAccess/VisualBasicQualifyMemberAccessCodeFixProvider.vb

```diff
@@ -0,0 +1,13 @@
+﻿'Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.CodeFixes.Qualify
+
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeFixProviderNames.QualifyMemberAccess), [Shared]>
+    <ExtensionOrder(After:=PredefinedCodeFixProviderNames.RemoveUnnecessaryCast)>
+    Friend Class VisualBasicQualifyMemberAccessCodeFixProvider
```

Comment:
We currently still require this for the `IsAlreadyQualifiedMemberAccess()` override, but once #7582 is fixed then C#/VB can be combined.